docs/deploy/howto-install-iot-edge.md

```diff
@@ -1,18 +1,26 @@
-# How to Install Azure IoT Edge for Industrial Scenarios
+# How to Setup a Local Gateway for Industrial Scenarios
```

Comment:
I thought we wanted to use IoT Edge instead of Gateway for consistency`?

Reply:
IoT Edge Installation is only a part of the gateway installation. Picking the gateway is also a part and so is setting it up in the cloud. Hence the overall title needed to change.